src/components/editMeasure/editor/MeasureEditor.tsx

```diff
@@ -383,6 +383,7 @@ const MeasureEditor = () => {
                 height="calc(100vh - 135px)"
                 readOnly={!canEdit}
                 setOutboundAnnotations={setOutboundAnnotations}
+                measureStoreCql={measure?.cql}
```

Comment:
Question: Does this use the `measure.cql` instead of `editorVal` so that it only processes against saved CQL?

Reply:
Yes, this only uses the actual measure cql as we are processing only when the cql is saved not when it changes